var/spack/repos/builtin/packages/python/package.py

```diff
@@ -759,51 +747,42 @@ def config_vars(self):
         if dag_hash not in self._config_vars:
             # Default config vars
             version = self.version.up_to(2)
-            try:
-                cc = self.compiler.cc
-                cxx = self.compiler.cxx
-            except (TypeError, NoCompilerForSpecError):
```

Comment:
Not worth all the error handling. For most installations, this will be overridden anyway.